### PR TITLE
Added docs explaining multiple --restrict-to-path flags, with example

### PR DESCRIPTION
**Pull request by icefoxen**
Updated docs to solve issue #1602, if you want them!

### CODE REVIEW

File: docs/usage/serve.rst.inc [hunk 1]
```diff
@@ -8,7 +8,9 @@ borg serve
 
 optional arguments
     ``--restrict-to-path PATH``
-        | restrict repository access to PATH
+        | restrict repository access to PATH.  Can be
+        | specified multiple times to allow the client
+        | access to several directories.
```
Comment (ThomasWaldmann) [inline docs/usage/serve.rst.inc]: you need to do this change in archiver.py - the *.inc files are created automatically from there.

File: docs/deployment.rst [hunk 2]
```diff
@@ -81,6 +85,7 @@ or relative remote repository path and can directly access the repositories at
 .. note:: The setup above ignores all client given commandline parameters
           which are normally appended to the `borg serve` command.
 
+
```
Comment (ThomasWaldmann) [inline docs/deployment.rst]: is this and accidental change or on purpose?

File: docs/deployment.rst [hunk 3]
```diff
@@ -53,6 +53,10 @@ Restrictions
 Borg is instructed to restrict clients into their own paths:
 ``borg serve --restrict-to-path /home/backup/repos/<client fqdn>``
 
+The client will be able to access any file or subdirectory inside of ``/home/backup/repos/<client fqdn>``
+but no other directories.  You can allow a client to access several directories by passing multiple
+`--restrict-to-path` flags, for instance: ``borg serve --restrict-to-path /home/backup/repos/<client fqdn>/root --restrict-to-path /home/backup/repos/<client fqdn>/home``
+
```
Comment (ThomasWaldmann) [inline docs/deployment.rst]: deployment.rst is just an example - should this be rather in the reference docs?

also, maybe mention that one can just create several repo subdirectories in there, so the client has multiple repos but just one --restrict-to-path.

File: docs/deployment.rst (hunk 2 above)
Comment (icefoxen) [inline docs/deployment.rst]: Accidental, sorry.

**Comment by ThomasWaldmann** [comment]
@icefoxen could you do these changes (see comments above)?

**Comment by icefoxen** [comment]
Yeah I'll get to it, sorry for the delay.  Hopefully tomorrow or this weekend.

**Comment by icefoxen** [comment]
Second attempt of this in PR #1676, committing against the right branch.  Sorry it took so long.

**Comment by ThomasWaldmann** [comment]
superceded by #1676

**Pull request closed without merging**